store/postgres/star_repository.go

```diff
@@ -0,0 +1,248 @@
+package postgres
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/odpf/columbus/asset"
+	"github.com/odpf/columbus/star"
+	"github.com/odpf/columbus/user"
+)
+
+// StarRepository is a type that manages star operation to the primary database
+type StarRepository struct {
+	client *Client
+}
+
+// Create insert a new record in the stars table
+func (r *StarRepository) Create(ctx context.Context, userID string, assetID string) (string, error) {
+	var starID string
+	if userID == "" || assetID == "" {
+		return "", star.InvalidError{
```

Comment:
For this kind of error, I'm thinking if it would be better to use concrete error instead 🤔 . If both userID and assetID are empty, we will not get any useful information from the error.

Example
```
package star

const (
  ErrEmptyUserID = errors.New("star is not related to any user")
  ErrEmptyAssetID = errors.New("star is not related to any asset")
)
```

wdyt?

Reply:
yes we can do that